src/utils/hooks/useLoadApplicationNEW.ts

```diff
@@ -82,13 +82,19 @@ const useLoadApplication = ({ serialNumber, networkFetch }: UseGetApplicationPro
         if (element.elementTypePluginCode === 'pageBreak') pageCount++
         else
           baseElements.push({
-            ...element,
+            // ...element,
             pluginCode: element.elementTypePluginCode,
             sectionIndex: sectionNode?.templateSection?.index,
             sectionCode: sectionNode?.templateSection?.code,
             elementIndex: element.index,
             page: pageCount,
-          } as TemplateElementState)
+            isEditableExpression: element.isEditable,
```

Comment:
looks like we need `code: element.code` (otherwise responses are not linked)

Reply:
Ah yeah, good catch. I was just eyeballing the previous structure output to check all the fields -- must have missed that one!

d682ddf